docs/concept/TAP7.1-solution-design.md

```diff
@@ -0,0 +1,425 @@
+
+# 3 AS-IS Analysis
+
+## 3.1 People
+
+### 3.1.1 Target Group Analysis
+
+See "4.1.1 To-be Target Group Analysis" below. No deviations between As-is and To-be target groups have been identified.
+
+## 3.2 Process
+
+### 3.2.1 Use Case View (high-level)
+
+Umbrella Chart is intended for the following Use Cases:
+
+| Actor             | Use case                                                                                             |
+|-------------------|------------------------------------------------------------------------------------------------------|
+| Product Developer | Sandbox Environments: Create local environments to evaluate and experiment with Catena-X components. |
+| Product Developer | Development: Provide a unified setup for contributors and developers working on Tractus-X projects.  |
+| Product Tester    | Testing: Run end-to-end tests for Catena-X services.                                                 |
+
+## 3.4 System
+
+### 3.4.1 Context Delimitation
+
+````mermaid
+flowchart
+    subgraph "Umbrella Chart"
+        ChartYaml["Chart.yaml"]
+        subgraph charts
+            Portal["Portal"]
+            CentralIdp["CentralIdp"]
+            DiscoveryFinder["Discovery Finder"]
+            SDFactory["SD-Factory"]
+            SSICredentialIssuer["SSI Credential Issuer"]
+            SemanticHub["Semantic Hub"]
+            BPDM["BPDM"]
+            DataConsumer["Data Consumer"]
+            DataProvider["Data Provider"]
+            BDRS["BDRS"]
+            SSIWalletStub["SSI Wallet Stub"]
+        end
+        subgraph "tx-data-provider"
+            EDC["EDC"]
+            DTR["DTR"]
+            SubmodelServer["Submodel Server"]
+        end
+    end
+
+    ChartYaml --> Portal
+    ChartYaml --> CentralIdp
+    ChartYaml --> DiscoveryFinder
+    ChartYaml --> SDFactory
+    ChartYaml --> SSICredentialIssuer
+    ChartYaml --> SemanticHub
+    ChartYaml --> BPDM
+    ChartYaml --> DataConsumer
+    ChartYaml --> DataProvider
+    ChartYaml --> BDRS
+    ChartYaml --> SSIWalletStub
+    DataConsumer --> EDC
+    DataProvider --> EDC
+    DataProvider --> DTR
+    DataProvider --> SubmodelServer
+````
+
+#### Existing Umbrella Chart Structure
+
+# 4 TO-BE Design
+
+## 4.1 People
+
+### 4.1.1 Target Group Analysis
+
+Overall data space roles: https://catena-x.net/en/1/catena-x-introduce-implement/how-to-operate-catena-x
+
+The relevant data space roles for us under requirement 7.1.1.1 are:
+
+- Data Consumer
+- Data Provider
+- Enablement Service Provider
+
+Scope: Umbrella HELM Charts
+
+- Enablement Services: These services are included under the umbrella of HELM charts, focusing on enabling
```

Comment:
This is still a mixup between in scope and out of scope. I have already addressed this with a comment.